src/features/surveys/components/surveyForm/SurveySubheading.tsx

```diff
@@ -0,0 +1,28 @@
+import { createElement, FC, ReactElement } from 'react';
+
+export type SurveySubheadingProps = {
+  children: ReactElement | string;
+  component?: string;
+};
+
+const SurveySubheading: FC<SurveySubheadingProps> = ({
+  children,
+  component = 'span',
+}) => {
+  return createElement(
```

Comment:
You could use `<Typography component="span">` for this, I believe.

Reply:
omfg it works! Honest to god I tried this an hour ago and got an error. The couple of coffees since then plus your comment have made it work somehow!

Reply:
Haha! Maybe you tried `variant="span"` and not `component="span"`? The former is probably not possible, but the latter is. The form controls style (and sets a default component based on the variant, I believe) while the latter just picks what HTML element to use.